examples/plotting.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"math"
+	"os"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+
+	"github.com/tobgu/qframe"
+	qfnum "github.com/tobgu/qframe/contrib/gonum"
+)
+
+func Sin(n int, max float64) qframe.QFrame {
+	xs := make([]float64, n)
+	ys := make([]float64, n)
+	for i := 0; i < n; i++ {
+		xs[i] = max / float64(n) * float64(i)
+		ys[i] = math.Sin(xs[i]) * 100
+	}
+	return qframe.New(map[string]interface{}{
+		"X": xs,
+		"Y": ys,
+	})
+}
+
+func main() {
+	plt, _ := plot.New()
+	qf := Sin(100, 2*math.Pi)
+	// Get the views for X and Y columns
+	xview, _ := qf.FloatView("X")
```

Comment:
error handling? (remember, it's an example that might be copy-pasted multiple times...)